### DISCUSSION TITLE
Use cases in a research lab

### DISCUSSION THREAD

**Discussion by carmelom** [inventree/InvenTree#8237] [Ideas]
Hi everyone,
I'm a PI of a newly-starting research lab in atomic physics... With lots of materials to buy from scratch: lasers, optomechanics, electronics, etc. I just found out about your project, and it feels it could really fit my use case. I'm definitely going to give it a try.

The project clearly fits a wide range of use cases, from hobbyists to production-ready companies, but how about any corner-case like mine in the research field? I'd like to hear from anyone else in a similar situation about their experience, tips and tricks or else.

Thanks!

**Comment by matmair** [comment]
I have heard of use in research institutions in electronics, mechatronics and bio anecodetialy at conferences although this is not something core InvenTree is optimised for.
If you run into bumps I would be very interested to hear about them either here or via mail (link in profile). There have been suggestions to add a tools section with things like safety/certification mandates but that has never been really hashed out.

**Reply by matmair** [reply]
@carmelom have you ended up using InvenTree? Always very interested in hearing from and supporting academic users

**Comment by westinpigott** [comment]
I use it for mechatronics.   Tracking the parts that come in from Amazon, McMaster, Digikey, Mouser, etc.  Its a bit manual to initially import the parts, but once they are in, its amazing.  We were drowning under our previous organization system, as everytime we got things "perfect" a new part would come along that didn't fit the schema.   We finally said "We don't need to keep similar parts together" and instead use inventree with its search to be able to find whatever we need.   Tells us what rack, shelf, bin, sub-bin, compartment, etc.  Works great!
